molecule/python/converge.yml

```
@@ -50,3 +50,16 @@
         assessment_tool_install_dir: /tools/mitm6
         assessment_tool_pip_packages:
           - mitm6
+    - name: Install the sqlmap package in a venv
```

Since it's not organized as a Python package and we do not create a venv.

```suggestion
    - name: Install the sqlmap tool
```